src/org/labkey/test/tests/SecurityTest.java

```diff
@@ -858,7 +861,8 @@ public void loginSelfRegistrationDisabledTest()
         assertFalse("Self-registration button is visible", link != null && link.isDisplayed());
 
         beginAt(buildURL("login", "register"));
-        assertElementPresent(Locators.labkeyError.withText("Registration is not enabled."));
+        waitForAnyElement(Locators.labkeyError, Locators.labkeyErrorSubHeading);
+        assertElementPresent(Locators.labkeyErrorSubHeading.withText("Registration is not enabled."));
```

Comment:
This check should be specific. We know what page we should end up on. If timing is a problem, we can wait for the error.
```suggestion
        waitForElement(Locators.labkeyErrorSubHeading.withText("Registration is not enabled."));
```